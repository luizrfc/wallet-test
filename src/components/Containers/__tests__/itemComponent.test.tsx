import React, { Fragment } from "react";
import "react-native";
import renderer from "react-test-renderer";
import Item from "../item";

test("Render item container component correctly", () => {
  const button = renderer
    .create(
      <Item type="input" mt={10}>
        <Fragment></Fragment>
      </Item>
    )
    .toJSON();
  expect(button).toMatchSnapshot();
});
