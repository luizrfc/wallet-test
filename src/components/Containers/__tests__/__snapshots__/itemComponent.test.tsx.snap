// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render item container component correctly 1`] = `
<View
  style={
    [
      {
        "width": "100%",
      },
      {
        "paddingBottom": 0,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 0,
      },
      {
        "marginBottom": 6,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 10,
      },
      {
        "alignItems": "center",
      },
    ]
  }
/>
`;
